dll/win32/msv1_0/msv1_0.c

```diff
@@ -1268,6 +1269,14 @@ LsaApLogonUserEx2(IN PLSA_CLIENT_REQUEST ClientRequest,
         Status = RtlValidateUnicodeString(0, &LogonInfo->UserName);
         if (!NT_SUCCESS(Status))
             return STATUS_INVALID_PARAMETER;
+
+        // ms docs says max length is 0xFF bytes.
+        // not in docs:
```

Comment:
Here it may be nice to quote one of the posts from
https://groups.google.com/forum/#!topic/microsoft.public.win32.programmer.kernel/eFGcCo_ZObk

Reply:
done